cmd/argoexec/commands/root.go

```diff
@@ -90,6 +91,9 @@ func initExecutor() *executor.WorkflowExecutor {
 	clientset, err := kubernetes.NewForConfig(config)
 	checkErr(err)
 
+	apiextensionClientSet, err := apiextensionclientset.NewForConfig(config)
```

Comment:
could we use `dynamic.NewForConfig` instead and save adding a new dependency?

Reply:
Yes, that's one option I thought about. The problem with `dynamic.Interface` is that `dynamic.Interface.Resource()` requires a `schema.GroupVersionResource` where we need to obtain the resource's plural name (which can be obtained by parsing the object's selfLink). Since we already have `k8s.io/apiextensions-apiserver` in `go.sum` so I thought the implementation would be cleaner if we just use `apiextensions-apiserver` directly.